components/admin_console/billing/payment_details.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useSelector} from 'react-redux';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+import CardImage from 'components/payment_form/card_image';
+import {GlobalState} from 'types/store';
+
+export interface PaymentDetailsProps {
+    children?: React.ReactNode;
+}
+
+const PaymentDetails: React.FC<PaymentDetailsProps> = ({children}: PaymentDetailsProps) => {
+    const paymentInfo = useSelector((state: GlobalState) => state.entities.cloud.customer);
```

Comment:
paymentInfo as the var name seems a little weird because we're actually getting the customer here. I'm not really suggesting to change it to `customer`,  but maybe add a comment or something?

Reply:
makes sense, renaming it to customerPaymentInfo will give more context from where is this information coming